app/components/nav_bar/user_component.html.erb

```diff
@@ -11,24 +11,33 @@
                 <%= inline_svg_tag "logo.svg", class: "h-10 w-auto", alt: "RailsDevs logo", title: "RailsDevs logo" %>
               <% end %>
             </div>
-
```

Comment:
Can you please add back in all of these deleted blank lines?

Reply:
fixed in [f09d2c3](https://github.com/joemasilotti/railsdevs.com/pull/818/commits/f09d2c3ec099818723bd5a229087f5d88b5340b5)